src/_reactComponents/PanelHeaderComponents/Checkbox.jsx

```diff
@@ -6,48 +6,52 @@ import {
 import React, { useRef } from 'react';
```

Comment:
Also creates weird margin on right side of component.
<img width="234" alt="Screen Shot 2022-05-15 at 1 33 26 PM" src="https://user-images.githubusercontent.com/68914928/168488469-c9b6eb2e-1492-4a9d-80bd-4e9b9cc7f2f6.png">
<img width="234" alt="Screen Shot 2022-05-15 at 1 35 30 PM" src="https://user-images.githubusercontent.com/68914928/168488553-95a433d5-0867-4afe-a632-e4abba5d0d41.png">